src/devices/vdml_imu.c

```diff
@@ -0,0 +1,144 @@
+/**
+ * \file devices/vdml_imu.c
+ *
+ * Contains functions for interacting with the VEX Inertial sensor.
+ *
+ * Copyright (c) 2017-2019, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <errno.h>
+#include "pros/imu.h"
+#include "v5_api.h"
+#include "vdml/registry.h"
+#include "vdml/vdml.h"
+
+#define ERROR_IMU_STILL_CALIBRATING(port, device, err_return)                  \
+	if (vexDeviceImuStatusGet(device->device_info) & E_IMU_STATUS_CALIBRATING) { \
```

Comment:
Would be nice to add an assert here that we're under the mutex, but I remember looking earlier and not seeing a way to do it.